source/fundamentals/encrypt-fields.txt

```diff
@@ -0,0 +1,73 @@
+==============
+Encrypt Fields
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can encrypt fields in a document using a feature called
+**client-side encryption**.
+
+Client-side encryption enables your client applications to encrypt data
+*before* sending it to MongoDB, and to query documents for values in
+encrypted fields.
+
+Because the driver encrypts the data before sending it to MongoDB, only
+your configured client applications can decrypt the data. Only applications
+using the driver with access to your encryption keys can access the decrypted,
+plaintext data. Should you have unauthorized access
+to your database, an attacker could only see the encrypted version of the
+fields you configured for encryption.
+
+Client-side encryption can prevent exposure of the following sensitive types of data:
+
+- Credit card numbers
+- Addresses
+- Health information
+- Financial information
+- Any other sensitive or personally identifiable information (PII)
+
+MongoDB offers the following ways to encrypt data:
+
+{+qe+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+{+qe+} is the next-generation client-side encryption feature,
+introduced in MongoDB 6.0. {+qe+} supports searching encrypted fields
+for equality and encrypts each value uniquely.
+
+The MongoDB manual contains detailed information on the following {+qe+} topics:
+
+- Begin a working {+qe+} example with the :ref:`qe-quick-start`
+- Learn how to use {+qe+} in the :ref:`qe-fundamentals`
+- Learn how to integrate your {+qe+} implementation with a KMS in the :ref:`qe-tutorials`
+- Learn {+qe+} concepts in the :ref:`qe-reference`
+
+{+csfle-long+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+{+csfle-long+} was introduced in MongoDB v4.2 and supports searching encrypted fields.
+{+csfle-short+} differs from {+qe+} in that it requires that encrypted fields
+you want to search must be determinstically encrypted. When you deterministically
+encrypt a value, the same input value produces the same output value.
+
+An unauthorized party who gains access to your data may be able to infer
```

Comment:
This is too strong a statement about leakage and inference because it is only fields with low cardinality that is a problem so I'd take some version of the wording from the deterministic description in the docs "While deterministic encryption provides greater support for read operations, encrypted data with low cardinality is susceptible to frequency analysis recovery."

Reply:
so i don't want an example in there either, just the statement about about low cardinality